src/openrct2/world/TileInspector.cpp

```diff
@@ -285,6 +288,68 @@ sint32 tile_inspector_paste_element_at(sint32 x, sint32 y, rct_tile_element elem
 
     if (flags & GAME_COMMAND_FLAG_APPLY)
     {
+        // Check if the element to be pasted refers to a banner index
+        uint8                     bannerIndex = BANNER_NULL;
+        const rct_scenery_entry * entry;
+        switch (tile_element_get_type(&element))
+        {
+        case TILE_ELEMENT_TYPE_WALL:
```

Comment:
We have `tile_element_get_banner_index()` for this sort of thing.

Reply:
I must have overlooked that function, thanks for pointing it out.